uiplib/gui/settingsTab.py

```diff
@@ -19,17 +19,34 @@ def create_settings_tab(appObj):
     # Add pics-folder setting
     pics_folder_label = Label(mainFrame, text="Where pics are stored:")
     pics_folder_label.grid(row=0, padx=10, pady=10, sticky=W)
+    appObj.pics_folder = StringVar()
     input_browse = Button(mainFrame,
                           text="Browse",
-                          command=lambda: file_open_helper(appObj))
+                          command=lambda: file_open_helper(appObj,
+                                                           appObj.pics_folder))
```

Comment:
Consider the redundancy here, you shouldn't pass something that it already owns? There's no point in doing that.